src/Address.svelte

```diff
@@ -100,7 +100,7 @@
         <Badge variant="foreground">org</Badge>
       {/if}
     {:else if addressType === AddressType.Contract}
-      <a href={explorerLink(address, wallet)} target="_blank" rel="noreferrer">
+      <a use:link href={`/${address}`}>
```

Comment:
?

Reply:
It's because we got rid of all the org stuff and now can't distinguish between orgs and other contracts. And since `gip-editors` is a contract it would link to etherscan otherwise. This change allows us to navigate to the profile of `gip-editors` within the app.


https://user-images.githubusercontent.com/158411/199675880-2eed4c9e-6ad8-4b13-80de-0d64ad016b6c.mov